core/src/mindustry/world/blocks/production/Drill.java

```diff
@@ -66,6 +67,19 @@ public Drill(String name){
         idleSoundVolume = 0.003f;
     }
 
+    @Override
+    public void drawRequestConfigTop(BuildRequest req, Eachable<BuildRequest> list){
+        Tile tile = req.tile();
+        if(tile == null) return;
+
+        countOre(req.tile());
+        if(returnItem == null) return;
+
+        Draw.color(returnItem.color);
+        Draw.rect(topRegion, req.drawx(), req.drawy(), 1f);
```

Comment:
Aren't you drawing it with 1 degree of rotation here?

Reply:
I didn't notice before as I copied it from `draw()`. Removing `1f` doesn't seem to change anything so I'll remove that and from `draw()`